com.unity.netcode.gameobjects/Editor/Simulator/NetworkScenarioView.cs

```diff
@@ -13,42 +13,87 @@ public class NetworkScenarioView : VisualElement
 
         readonly NetworkScenario m_NetworkScenario;
         readonly List<Type> m_Scenarios;
+        readonly SerializedProperty m_NetworkScenarioProperty;
+        readonly List<string> m_Choices;
 
-        DropdownField PresetDropdown => this.Q<DropdownField>(nameof(PresetDropdown));
+        DropdownField ScenarioDropdown => this.Q<DropdownField>(nameof(ScenarioDropdown));
 
-        public NetworkScenarioView(NetworkScenario networkScenario)
+        public NetworkScenarioView(SerializedProperty networkScenarioProperty, NetworkScenario networkScenario)
         {
             m_NetworkScenario = networkScenario;
+            m_NetworkScenarioProperty = networkScenarioProperty;
 
             AssetDatabase.LoadAssetAtPath<VisualTreeAsset>(UXML).CloneTree(this);
+            
+            // Using IMGUIContainer since the PropertyField doesn't properly support SerializedReferences until Unity 2022.1
```

Comment:
I wonder if @ian-m-unity knows about this, he had a ton of trouble with SerializedReferences in RNSM